Pod/Classes/PINRemoteImageCategoryManager.m

```diff
@@ -172,7 +172,7 @@ + (void)setImageOnView:(id <PINRemoteImageCategory>)view
     if (placeholderImage) {
         [view pin_setPlaceholderWithImage:placeholderImage];
     }
-    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsNone;
+    PINRemoteImageManagerDownloadOptions options = [view pin_defaultOptions];
```

Comment:
Yup, my point is it would break any subclasses which conform with the protocol. The reason it's a protocol is to specifically allow categories outside the library :) For example, we have a category to couple AsyncDisplayKit's ASImageNode to PINRemoteImage.